module/zfs/dmu_object.c

```diff
@@ -341,4 +384,10 @@ EXPORT_SYMBOL(dmu_object_free);
 EXPORT_SYMBOL(dmu_object_next);
 EXPORT_SYMBOL(dmu_object_zapify);
 EXPORT_SYMBOL(dmu_object_free_zapified);
+
+/* BEGIN CSTYLED */
+module_param(dmu_object_alloc_chunk_shift, int, 0644);
+MODULE_PARM_DESC(dmu_object_alloc_chunk_shift,
+	"CPU-specific allocator grabs 2^N objects at once");
```

Comment:
Are you sure it's safe to allow this to be changed at run time?  This would result in a miscalculation of where the per-cpu blocks of objects ends.  Although it looks like that would result in some wasted called to `dnode_hold_impl()` and then it would sort itself out.  Right?